### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by petered**
Propose adding some "getting started" details that were not obvious to me at first.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,8 +43,12 @@ dependencies {
 }
 ```
 
+If using Android Studio, you may need to go `File -> Invalidate Caches -> Restart` in order for the IDE to recognise MultiK.
```
Comment (AlexandreBrown) [inline README.md:46]: If it's true for Android Studio then I believe it would be true for any Intellij IDEA not just Android Studio.  
What do you think?

Comment (petered) [reply README.md:46]: Probably, can't confirm as I've only used Android Studio

Comment (AlexandreBrown) [reply README.md:46]: The steps you described are available on any Intellij IDEA. In fact I believe Android Studio was developed with the same IDEA base that is found in every intellij IDEA (it was built on top).

Might be beneficial to be more broad in the description or to mention "Android Studio or Intellij IDEA"

**Comment by devcrocod** [comment]
What versions is your Android Studio and Kotlin plugin?

**Pull request closed without merging**